linux_os/guide/services/xwindows/disabling_xwindows/xwindows_runlevel_target/oval/shared.xml

```diff
@@ -0,0 +1,49 @@
+{{%- if init_system == "systemd" and target_oval_version == [5, 10] -%}}
+{{# this is the only scenario this definition cannot handle, there is no good alternative for symlink_test for OVAL 5.10 #}}
+{{%- else -%}}
+<def-group>
+  <definition class="compliance" id="xwindows_runlevel_target" version="1">
+    <metadata>
+      <title>Disable X Windows Startup By Setting Default SystemD Target</title>
+      {{{- oval_affected(products) }}}
+      {{%- if init_system == "systemd" %}}
+      <description>Checks /etc/systemd/system/default.target to ensure that the default runlevel target is set to multi-user.target.</description>
+      {{%- else %}}
+      <description>Checks /etc/inittab to ensure that default runlevel is set to 3.</description>
+      {{%- endif %}}
+    </metadata>
+    {{%- if init_system == "systemd" %}}
+    <criteria>
+      <criterion comment="default.target systemd softlink exists" test_ref="test_disable_xwindows_runlevel_target" />
+    </criteria>
+    {{%- else %}}
+    <criteria>
+      <criterion comment="default runlevel is 3" test_ref="test_etc_inittab_default_runlevel_target" />
+    </criteria>
+    {{%- endif %}}
+  </definition>
+  {{%- if init_system == "systemd" %}}
+  <unix:symlink_test check="all" check_existence="all_exist" comment="default.target systemd softlink exists" id="test_disable_xwindows_runlevel_target" version="1">
+    <unix:object object_ref="object_disable_xwindows_runlevel_target" />
+    <unix:state state_ref="state_disable_xwindows_runlevel_target" />
+  </unix:symlink_test>
+  <unix:symlink_object comment="default.target systemd softlink exists" id="object_disable_xwindows_runlevel_target" version="1">
+    <unix:filepath>/etc/systemd/system/default.target</unix:filepath>
+  </unix:symlink_object>
+  <unix:symlink_state comment="default.target is linked to multi-user.target" id="state_disable_xwindows_runlevel_target" version="1">
+    <unix:filepath>/etc/systemd/system/default.target</unix:filepath>
+    <unix:canonical_path operation="pattern match">^/usr/lib/systemd/system/multi-user.target$</unix:canonical_path>
+  </unix:symlink_state>
+  {{%- else %}}
+  <ind:textfilecontent54_test check="all" comment="default runlevel is 3" id="test_etc_inittab_default_runlevel_target" version="1">
+    <ind:object object_ref="object_etc_inittab_default_runlevel_target" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object id="object_etc_inittab_default_runlevel_target" version="1">
+    <ind:filepath>/etc/inittab</ind:filepath>
+    <ind:pattern operation="pattern match">^[\s]*id:3:initdefault:[\s]*$</ind:pattern>
```

Comment:
This shouldn't be here. It should stay under `xwindows_runlevel_setting`

Reply:
agreed